pkg/util/version/const.go

```diff
@@ -27,21 +27,21 @@ var GitCommit = "unknown"
 
 // InstallStream describes stream we are defaulting to for all new clusters
 var InstallStream = &Stream{
-	Version:  NewVersion(4, 8, 10),
-	PullSpec: "quay.io/openshift-release-dev/ocp-release@sha256:53576e4df71a5f00f77718f25aec6ac7946eaaab998d99d3e3f03fcb403364db",
+	Version:  NewVersion(4, 8, 11),
```

Comment:
Why is this change part of the PR? Wasn't it done by Matt a couple weeks ago in a separate PR?

Reply:
Nothing to see here :eyes: (bad rebase :upside_down_face:)